material-popup-menu/src/main/java/com/github/zawadz88/materialpopupmenu/internal/PopupMenuAdapter.kt

```diff
@@ -123,9 +126,9 @@ internal class PopupMenuAdapter(
     ) : AbstractItemViewHolder(itemView) {
 
         override fun bindItem(popupMenuItem: MaterialPopupMenu.AbstractPopupMenuItem) {
+            super.bindItem(popupMenuItem)
             val popupMenuCustomItem = popupMenuItem as MaterialPopupMenu.PopupMenuCustomItem
             popupMenuCustomItem.viewBoundCallback.dismissPopupAction = dismissPopupCallback
```

Comment:
This should be moved together with a call to invoke so the dismiss action will also work for other classes that extend `AbstractItemViewHolder`.

Reply:
Nice catch!